backend/pkg/config/config.go

```diff
@@ -154,6 +155,7 @@ func flagset() *flag.FlagSet {
 	f.Bool("dev", false, "Allow connections from other origins")
 	f.Bool("insecure-ssl", false, "Accept/Ignore all server SSL certificates")
 	f.Bool("enable-dynamic-clusters", false, "Enable dynamic clusters, which stores stateless clusters in the frontend.")
+	f.Bool("watch-plugins-changes", true, "Reloads plugins when there are changes to them or their directory")
```

Comment:
Make it explicit here that the value will be false by default if -in-cluster is true.